tests/integration/test_encoding.py

```diff
@@ -63,3 +63,22 @@ def test_trace_with_metrics_accepted_by_agent(self, metrics):
             tracer.shutdown()
         log.warning.assert_not_called()
         log.error.assert_not_called()
+
+    @pytest.mark.parametrize(
+        "span_links_kwargs",
+        [
+            {"trace_id": 12345, "span_id": 67890},
+        ],
+    )
+    def test_trace_with_links_accepted_by_agent(self, span_links_kwargs):
+        """The agent does not return an error code when accepting a trace payload containing span links."""
+        tracer = Tracer()
+        with mock.patch("ddtrace.internal.writer.writer.log") as log:
+            with tracer.trace("root", service="test_encoding", resource="test_resource") as root:
+                root.set_link(**span_links_kwargs)
+                for _ in range(999):
```

Comment:
nit: Do we need this to iterate this many times? Not a huge deal, but wondering if there was a reason to make it generate that many traces

Reply:
probably not, i was just following the existing pattern

Reply:
i should probably rebase this so i'll change the number while i'm at it.

Reply:
changed it to 10